package-lock.json

```diff
@@ -4059,7 +4059,7 @@
     },
     "load-json-file": {
```

Comment:
Can you exclude all changes to this file from your commit?

I know it's not you. It's just NPM being shitty and inconsistent between different versions. But still, it shouldn't be here.